ladybug_geometry/polyskel.py

```diff
@@ -0,0 +1,558 @@
+# coding=utf-8
+"""
+Implementation of the straight skeleton algorithm as described by Felkel and Obdržálek in their 1998 conference paper Straight skeleton implementation.
+"""
+
+from __future__ import division
+
+import logging
+import heapq
+from itertools import *
+from collections import namedtuple
+import operator
+
+# Geometry classes
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry2d.pointvector import Point2D, Vector2D
+from ladybug_geometry.geometry2d.line import LineSegment2D
+from ladybug_geometry.geometry2d.ray import Ray2D
+from ladybug_geometry import intersection2d
+
+log = logging.getLogger("__name__")
+
+
+# For Debugging
+
+class Debug:
```

Comment:
If this is only for development then you probably want to name it as `_Debug` so it doesn't show up in the documentation.

Reply:
Done